\

```diff
@@ -0,0 +1,245 @@
+# encoding: UTF-8
+require "logstash/devutils/rspec/spec_helper"
+require "logstash/outputs/file"
+require "logstash/event"
+require "logstash/json"
+require "stud/temporary"
+require "tempfile"
+require "uri"
+
+describe LogStash::Outputs::File do
+  describe "ship lots of events to a file" do
+    tmp_file = Tempfile.new('logstash-spec-output-file')
+    event_count = 10000 + rand(500)
+
+    config <<-CONFIG
+    input {
+      generator {
+        message => "hello world"
+        count => #{event_count}
+        type => "generator"
+      }
+    }
+    output {
+      file {
+        path => "#{tmp_file.path}"
+      }
+    }
+    CONFIG
+
+    agent do
+      line_num = 0
+      
+      # Now check all events for order and correctness.
+      tmp_file.each_line do |line|
+        event = LogStash::Event.new(LogStash::Json.load(line))
+        insist {event["message"]} == "hello world"
```

Comment:
no need to use insist here ;-), you can use the Rspec expect command ;-) https://relishapp.com/rspec/rspec-expectations/v/2-14/docs